apps/src/templates/sectionAssessments/StudentAssessmentOverviewTable.jsx

```diff
@@ -0,0 +1,200 @@
+import React, {Component, PropTypes} from 'react';
+import {Table, sort} from 'reactabular';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import i18n from '@cdo/locale';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import styleConstants from "@cdo/apps/styleConstants";
+import MultipleChoiceAnswerCell from './MultipleChoiceAnswerCell';
+import {
+  studentAnswerDataPropType,
+  questionDataPropType
+} from './assessmentDataShapes';
+
+export const COLUMNS = {
+  QUESTION: 0,
+  STUDENT_ANSWER: 1,
+  CORRECT_ANSWER: 2,
+};
+
+const ANSWER_COLUMN_WIDTH = 80;
+const PADDING = 20;
+const MAX_WIDTH = styleConstants['content-width'] - (5 * (ANSWER_COLUMN_WIDTH + PADDING));
+
+const styles = {
+  answerColumnHeader: {
+    width: ANSWER_COLUMN_WIDTH,
+    textAlign: 'center',
+  },
+  answerColumnCell: {
+    width: ANSWER_COLUMN_WIDTH,
+  },
+  questionCell: {
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+  }
+};
+
+class StudentAssessmentOverviewTable extends Component {
+  static propTypes= {
+    questionAnswerData: PropTypes.arrayOf(questionDataPropType),
+    studentAnswerData: PropTypes.arrayOf(studentAnswerDataPropType)
+  };
+
+  state = {
+    [COLUMNS.NAME]: {
+      direction: 'desc',
+      position: 0
+    }
+  };
+
+  getSortingColumns = () => {
+    return this.state.sortingColumns || {};
+  };
+
+  onSort = (selectedColumn) => {
+    this.setState({
+      sortingColumns: sort.byColumn({
+        sortingColumns: this.state.sortingColumns,
+        // Custom sortingOrder removes 'no-sort' from the cycle
+        sortingOrder: {
+          FIRST: 'asc',
+          asc: 'desc',
+          desc: 'asc'
+        },
+        selectedColumn
+      })
+    });
+  };
+
+  getCorrectAnswer = (multipleChoiceAnswers) => {
+    return multipleChoiceAnswers.filter(answerObj => {
+      return answerObj.isCorrectAnswer;
+    }).map(answerObj => {
+      return answerObj.multipleChoiceOption;
+    }).join(', ');
+  };
+
+  correctAnswerColumnFormatter = (answers, {rowData, columnIndex}) => {
+    const multipleChoiceAnswers = rowData.answers;
+
+    let textAnswered = this.getCorrectAnswer(multipleChoiceAnswers);
+
+    return (
+      <MultipleChoiceAnswerCell
+        id={rowData.id}
+        textAnswered={textAnswered}
+      />
+    );
+  };
+
+  studentAnswerColumnFormatter = (studentAnswers, {rowData, rowIndex}) => {
+    const selectStudentAnswers = this.props.studentAnswerData[0].studentAnswers[rowIndex];
+    const multipleChoiceAnswers = rowData.answers;
+
+    let studentResponse = selectStudentAnswers.answer.map(answerSelection => {
```

Comment:
rename `answer` on `selectStudentAnswers` to `answers` since it's an array of answers

on this same line, what are you mapping? currently, your map expression returns the same as `selectStudentAnswers.answer`, which i think is what you want. if so, you can remove `.map`

Reply:
Fixed: I also changed answer to answers in the studentData object in sectionAssessments file.